prompt-service/src/unstract/prompt_service/helper.py

```diff
@@ -80,3 +81,55 @@ def plugin_loader(app: Flask) -> dict[str, dict[str, Any]]:
         app.logger.info("No plugins found.")
 
     return plugins
+
+
+def query_usage_details(db, run_id, token):
+    org_id = AuthenticationMiddleware.get_account_from_bearer_token(token)
+    metadata = {"run_id": run_id}
+    query = f"""
+        SELECT
+            usage_type,
+            llm_usage_reason,
+            model_name,
+            SUM(prompt_tokens) AS input_tokens,
+            SUM(completion_tokens) AS output_tokens,
+            SUM(total_tokens) AS total_tokens,
+            SUM(embedding_tokens) AS embedding_tokens
+        FROM "{org_id}"."token_usage"
+        WHERE run_id = %s
```

Comment:
We should make an index on the column `run_id` if this query happens frequently. otherwise the query/transaction will take more memory,cpu and time. Depends on the size of table